src/Microsoft.Extensions.Logging/ILoggerBuilder.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Logging
+{
+    public interface ILoggerBuilder
```

Comment:
Why introduce `ILoggerBuilder` at all, here? If it's just a wrapper for `IServiceCollection`, and all the provider extension methods do is dot through to `Services`, isn't this unnecessary ceremony?